app/services/ComplianceService.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import connectors.ComplianceConnector
+import javax.inject.Inject
+import models.compliance.CompliancePayload
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.Future
+
+class ComplianceService @Inject()(connector: ComplianceConnector) {
+
+  def getComplianceDataWithVrn(enrolmentKey: String)(implicit hc: HeaderCarrier): Future[CompliancePayload] = connector.getComplianceData(enrolmentKey)
```

Comment:
This method name will probably need to change as its queried with an enrolmentKey not a VRN